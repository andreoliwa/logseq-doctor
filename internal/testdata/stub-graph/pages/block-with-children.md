- First item
- Parent block with children
  - Child 1
  - Child 2
- Last item
