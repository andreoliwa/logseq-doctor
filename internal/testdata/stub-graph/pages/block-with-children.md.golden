- First item
- Parent block with children
	- Child 1
	- Child 2
	- TODO Clean the room
- Last item
